api/bazel/repository_locations.bzl

```diff
@@ -90,9 +90,9 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "Prometheus client model",
         project_desc = "Data model artifacts for Prometheus",
         project_url = "https://github.com/prometheus/client_model",
-        version = "147c58e9608a4f9628b53b6cc863325ca746f63a",
-        sha256 = "f7da30879dcdfae367fa65af1969945c3148cfbfc462b30b7d36f17134675047",
-        release_date = "2021-06-07",
+        version = "63fb9822ca3ba7a4ba5184071fb8f2ea000a99ef",
```

Comment:
this has tagged versions (altho not actual releases) - https://github.com/prometheus/client_model/tags - should we use that ?